digitaltwins/samples/iot_central_device/sample_device.js

```diff
@@ -33,9 +33,9 @@ async function main() {
   const registrationResult = await provisioningClient.register();
   const deviceConnectionString = `HostName=${registrationResult.assignedHub};DeviceId=${registrationResult.deviceId};SharedAccessKey=${symmetricKey}`;
 
-  const propertyUpdateHandler = (interfaceInstance, propertyName, reportedValue, desiredValue, version) => {
+  const propertyUpdateHandler = (component, propertyName, reportedValue, desiredValue, version) => {
     console.log('Received an update for ' + propertyName + ': ' + JSON.stringify(desiredValue));
-    interfaceInstance[propertyName].report(desiredValue, {
+    component[propertyName].report(desiredValue, {
```

Comment:
>component[propertyName].report(desiredValue, { [](start = 4, length = 46)

Gonna have to change this to the client based rather than the component based.